components/org.wso2.identity.event.websubhub.publisher/src/main/java/org/wso2/identity/event/websubhub/publisher/constant/WebSubHubAdapterConstants.java

```diff
@@ -98,29 +119,44 @@ public enum ErrorMessages {
                 "Server error encountered while preparing SSL context for WebSubHub http client."),
         ERROR_CREATING_ASYNC_HTTP_CLIENT("65008", "Error while creating the Async HTTP client.",
                 "Server error encountered while creating the Async HTTP Client of WebSub Hub Adapter."),
-        TOPIC_DEREGISTRATION_FAILURE_ACTIVE_SUBS("65009", "Error occurred while de-registering topic", "Backend error" +
-                " received from WebSubHub while attempting to de-register topic: %s. Active subscribers: %s.");
+        TOPIC_DEREGISTRATION_FAILURE_ACTIVE_SUBS("65009", "Error occurred while de-registering topic",
+                "Backend error" +
+                " received from WebSubHub while attempting to de-register topic: %s. Active subscribers: %s."),
```

Comment:
Do we want to say 'Backend error' ?
Let's just say WebSubHub returned an error response